lib/benchmark/src/benchmark.cc

```diff
@@ -390,8 +390,6 @@ std::unique_ptr<BenchmarkReporter> CreateReporter(
     return PtrType(new ConsoleReporter(output_opts));
   } else if (name == "json") {
     return PtrType(new JSONReporter);
-  } else if (name == "csv") {
-    return PtrType(new CSVReporter);
```

Comment:
I don't think we have yet patched benchmark.cc. Did you check if this is still an issue in v1.8.0?

Reply:
No, but as war as I can see, we doesen't use this deprecated output format. Do you see a problem to remove this ununsed code lines?

Reply:
No, but it is a maintenance burden whenever we upgrade the library. Did you consider to just remove the warning in this external library?